app/src/main/java/fr/groggy/racecontrol/tv/ui/channel/ChannelCardPresenter.kt

```diff
@@ -34,15 +36,16 @@ class ChannelCardPresenter: Presenter() {
     }
 
     override fun onBindViewHolder(viewHolder: ViewHolder, item: Any) {
+        val resources = RaceControlTvApplication.context?.resources
```

Comment:
Btw, for future...

You can always retrieve the context here via `viewHolder.view.context`

Reply:
I tried that first, but apparently it is getting system resources and not app resources or something like that... Anyway, that didn't work for me.